packages/web/src/components/notification-dot/NotificationDot.tsx

```diff
@@ -0,0 +1,24 @@
+import { ComponentProps } from 'react'
+
+import cn from 'classnames'
+
+import styles from './NotificationDot.module.css'
+
+type NotificationDotVariant = 'small' | 'large'
+type NotificationDotProps = {
+  variant: NotificationDotVariant
+} & ComponentProps<'span'>
+
+/** Renders a basic notification dot used to decorate controls and indicate new information */
+export const NotificationDot = (props: NotificationDotProps) => {
```

Comment:
do we not have this dot already? just in mobile i guess?

Reply:
Just in mobile, correct. We had a sort of similar dot for when playlists are updated on desktop. But it was messy. This is formalizing it as a reusable component.